src/Main.java

```diff
@@ -67,20 +67,22 @@ public static void main(String[] args) {
         char[] yellow = {'y', 'e', 'l', 'l', 'o', 'w'};
         char[] orange = {'o', 'r', 'a', 'n', 'g', 'e'};
         char[] black = {'b', 'l', 'a', 'c', 'k'};
-        Museum m1 = new Museum(yellow,  2.0, 1.05, 287);
-        Museum m2 = new Museum(orange,  2.3,  1.06, 290);
-        Museum m3 = new Museum(black,  2.5,  1.07, 300);
 
-        Museum[] museums = new Museum[3];
-        museums[0] = m1;
-        museums[1] = m2;
-        museums[2] = m3;
+        Bursztyn b1 = new Bursztyn(yellow, 2.0, 1.05, 287);
+        Bursztyn b2 = new Bursztyn(orange, 2.3, 1.06, 290);
+        Bursztyn b3 = new Bursztyn(black, 2.5, 1.07, 300);
+
+        Bursztyn[] bursztyn = new Bursztyn[3];
+
+        bursztyn[0] = b1;
+        bursztyn[1] = b2;
+        bursztyn[2] = b3;
 
         //wyświetlenie temp. w Fahrenheit:
-        museums[1].displayF();
+        bursztyn[1].displayF();
 
         //wyświetlenie zwiększonej gęstości:
-        double higherDensity = museums[1].higherDensity();
-        System.out.println("Zwiększona gęstość wybranego bursztynu: "+higherDensity);
+        double higherDensity = bursztyn[1].higherDensity(0.02);
+        System.out.println("Zwiększona gęstość wybranego bursztynu: " + higherDensity);
     }
 }
```

Comment:
miałam jakiś konflikt w Main.java, który rozwiązałam, ale przez to ostatnie zmiany sa nieczytelne.